glidertools/utils.py

```diff
@@ -168,7 +168,7 @@ def calc_glider_vert_velocity(time, depth):
     return velocity
 
 
-def calc_dive_phase(time, depth):
+def calc_dive_phase(time, depth, dive_depth_threshold=15):
```

Comment:
Should the default be set to the previously hardcoded value? That way we avoid breaking workflows with old versions?

Reply:
Possibly. This is perhaps a question for the wider project. If we have a non-sensible default, either because it was selected arbitrary, or best practices suggest it should be x, should we preserve it and associated workflows, which would require all users to explicitly change it to follow a best practice, or do we make workflow breaking changes?
I'm not suggesting "15 m" is magically best practice but it does at least work for our test cases.

Personally I think defaults are very important and if we have a good reason to change them then we should, even if it breaks a workflow, especially while we are still adding lots of features.

Reply:
I am not a domain expert here (which might show 😆). If you think this leads to undesirable behavior, I am all for changing this. I totally agree on your point about defaults. The only thing that would be necessary is to add an entry in `whats-new.org` indicating that these will be 'Breaking Changes', which will then lead us to publish a new major version on the next release. Does that sound good?

Reply:
Maybe @tjryankeogh can also feedback on this?

Reply:
I agree that 15 m is a more useful default value. I work with Glider data from the Baltic Sea, and the sea floor (aka lower inflexion) is usually no deeper than 70 m. Also very nice to have this as a user defined parameter!